scripts/mkcloud

```diff
@@ -109,7 +109,7 @@ iscloudver 7plus && : ${controller_node_memory:=12582912}
 : ${lonelynode_hdd_size:=20}
 : ${ironicnode_hdd_size:=20}
 
-if [[ $hacloud ]] && [[ "$want_database_sql_engine" != "mysql" ]]; then
+if [[ $hacloud ]] && [[ "$want_database_sql_engine" == "postgresql" ]]; then
```

Comment:
As you noted already this doesn't work on Cloud 7 GA, when  "$want_database_sql_engine" is not set. I'd be fine with fixing this in a followup.

But I think we also still care about Cloud6, don't we?